### ISSUE TITLE
Ember.observer deprecation with ember 1.13.5

### ISSUE THREAD

**Issue by mike-post** [emberjs/data#3574]
After upgrading to ember 1.13.5, ember-data 1.13.6 gives the following deprecation warning:

````
DEPRECATION: Passing the dependentKeys after the callback function in Ember.observer is deprecated. Ensure the callback function is the last argument.
        at Object.observer (http://localhost:4200/assets/vendor.js:27515:34)
        at http://localhost:4200/assets/vendor.js:75373:27
        at http://localhost:4200/assets/vendor.js:83902:4
````

It looks like this fixed in commit 8fb11668dad8119dae45936491c11fec4dca8df9 (which is on master), but that commit did not make it into the 1.13.6 release.

**Comment by rwjblue** [comment]
@mike-post - Thanks for reporting! I saw @bmac mention this earlier and I believe that a 1.13.7 will be done in the next couple days to fix...

**Comment by denzo** [comment]
Where can I find more info on Ember.observer deprecation?

**Comment by rwjblue** [comment]
@denzo - Generally, the [CHANGELOG.md](https://github.com/emberjs/ember.js/blob/v1.13.5/CHANGELOG.md) lists all the changes for a given version (at least I try to make sure that it does).  It has this to say:

- [#11778](https://github.com/emberjs/ember.js/pull/11778) [DEPRECATION] Deprecate reverse argument ordering in `Ember.observer`.

That PR added the deprecation, and includes an example of the deprecated behavior.

---

**tldr:** 

Deprecated:

```javascript
Ember.observer(function() {
  // your stuff here
}, 'depKey1', 'depKey2');
```

Not deprecated:

```javascript
Ember.observer('depKey1', 'depKey2', function() {
  // your stuff here
});
```

**Comment by igorT** [comment]
https://github.com/emberjs/data/issues/3584 is the issue about making sure this makes it to a .13 point release

**Comment by ksnyde** [comment]
Hmmm ... I'm still getting this and am on `1.13.6. Stack:

>DEPRECATION: Passing the dependentKeys after the callback function in Ember.observer is deprecated. Ensure the callback function is the last argument.
        at observer (http://localhost:4200/assets/vendor.js:29044:34)
        at <anonymous>:3110:23
        at requireModule (<anonymous>:40:28)
        at requireModule (<anonymous>:36:24)
        at <anonymous>:6629:62
        at triggerOnce (<anonymous>:6661:7)
        at Object.runLoadHooks (http://localhost:4200/assets/vendor.js:48665:14)
        at http://localhost:4200/assets/vendor.js:13980:32
        at internalRequire (http://localhost:4200/assets/vendor.js:10636:16)

In fact I'm coding an addon at the moment so I'm pretty sure the 

> "ember-disable-prototype-extensions": "^1.0.0",

would block my code from using the aforementioned syntax. Wouldn't it?

**Comment by bmac** [comment]
@ksnyde I believe the issue was fixed in Ember Data 1.13.7.

**Comment by ksnyde** [comment]
Ok, just had a look and it's apparently not on bower yet. Will that be available soon?

**Comment by bmac** [comment]
Hmm, Thats strange I tought it was already on bower. https://github.com/components/ember-data/releases

I'll look into it.

**Comment by bmac** [comment]
@ksnyde I just created a new repo and updated the Ember Data entry in bower.json to 1.13.7 and I didn't have any issues installing the new version. Let me know if I'm missing something.

https://github.com/bmac/ember-data-test

**Comment by ksnyde** [comment]
oh I'm sorry, I thought you were referring to ember not ember-data. I've actually removed all ember-data from this addon so it shouldn't be the source. Now I see that my google search landed me into the ember data repo. Oops.
